modules/kube-audit/scripts/confirm-rollout-status.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+export PATH=$PATH:"/tmp"
```

Comment:
we should explain in a code comment why this is being done (same in all places we added this)